aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -126,10 +126,13 @@ def _get(self, name: str, **sdk_options) -> str:
         return_value = response["Configuration"].read()
         self._next_token[name] = response["NextPollConfigurationToken"]
 
+        # The return of get_latest_configuration can be null because this value is supposed to be cached
+        # on the customer side. We created a dict with the last returned value for the specific configuration
+        # See https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/appconfigdata/client/get_latest_configuration.html
```

Comment:
Add one line about the dict change.

This will help trace back this bug, along with a PR Body update to make it clear what the bug was